byon_service/exceptions.py

```diff
@@ -0,0 +1,65 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+
+
+class ByonHTTPException(Exception):
```

Comment:
Don't you think it would be little inconsistent if we leave HostPool in BYON repository? I mean it would be easier and cleaner if we do that after migration to the new repository and then change all names with one commit.